Userland/Utilities/hexdump.cpp

```diff
@@ -12,6 +12,12 @@
 
 static constexpr size_t LINE_LENGTH_BYTES = 16;
 
+enum class STATES {
```

Comment:
This should be named like a class (i.e. `States`), and since a value of this type can only have one of these enumerations, please call it `State`.